app/src/pages/inside/testCaseLibraryPage/expandedOptions/createFolderModal/createFolderModal.tsx

```diff
@@ -92,14 +111,54 @@ const CreateFolderModalComponent = ({
 
   const hideModal = () => dispatch(hideModalAction());
 
-  const onSubmit = (values: CreateFolderFormValues) => {
-    const idFromNameInput = coerceToNumericId(values.parentFolderName);
+  const handleSelectedFolder = ({ selectedItem }: { selectedItem: FolderWithFullPath }) => {
+    change('parentFolderName', selectedItem?.id || '');
+  };
+
+  const mappedFolders = useMemo(() => traverseFolders(buildFoldersMap(folders)), [folders]);
+
+  const renderOption = (
+    option: FolderWithFullPath,
+    index: number,
+    _isNew: boolean,
+    getItemProps: ({
+      item,
+      index,
+    }: {
+      item: FolderWithFullPath;
+      index: number;
+    }) => HtmlHTMLAttributes<HTMLDivElement>,
+  ) => (
+    <AutocompleteOption
+      isActive={true}
+      isSelected={true}
```

Comment:
You can simplify such code
```suggestion
      isActive
      isSelected
```